forc-test/src/lib.rs

```diff
@@ -543,34 +561,66 @@ pub enum TestRunnerCount {
     Auto,
 }
 
+pub struct TestCount {
+    pub total: usize,
+    pub filtered: usize,
```

Comment:
nit: consider renaming to `ignored` or `filtered_out`. I assumed this was the number of tests remaining after filtering, but it looks like this is the count of tests filtered out.

```suggestion
    pub ignored: usize,
```

Reply:
Ahh yeah missed this one, I actually changed the local vars name but forgot to apply it to the struct.